NSubstitute.DbConnection/NSubstitute.DbConnection/QueryParameter.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NSubstitute.DbConnection
+{
+    internal class QueryParameter
+    {
+        public QueryParameter(string name, object value)
+        {
+            Name = name;
+            Value = value;
+        }
+
+        public QueryParameter(string name, object value, bool output)
+        {
+            Name = name;
+            Value = value;
+            Output = output;
```

Comment:
By convention for boolean params / properties, I would call these `isOutput` and `IsOutput` respectively